frontend/javascripts/oxalis/view/right-border-tabs/export_bounding_box_modal.js

```diff
@@ -119,24 +126,30 @@ const ExportBoundingBoxModal = ({ handleClose, dataset, boundingBox, tracing }:
     };
   });
 
-  const exportButtonsList = allLayerInfos.map(layerInfos =>
-    layerInfos ? (
+  const exportButtonsList = allLayerInfos.map(layerInfos => {
+    const parenthesesInfos = [
+      startedExports.includes(exportKey(layerInfos)) ? "started" : null,
+      layerInfos.mappingName != null ? `using mapping "${layerInfos.mappingName}"` : null,
+      !layerInfos.hasMag1 ? "resolution 1 missing" : null,
+    ].filter(el => el);
+    const parenthesesInfosString =
+      parenthesesInfos.length > 0 ? ` (${parenthesesInfos.join(", ")})` : "";
+    return layerInfos ? (
       <p key={exportKey(layerInfos)}>
         <Button
           onClick={() => handleStartExport(layerInfos)}
           disabled={
             startedExports.includes(exportKey(layerInfos)) ||
             !layerInfos.hasMag1 ||
-            (isMergerModeEnabled && !layerInfos.isColorLayer)
+            (isMergerModeEnabled && layerInfos.tracingId != null)
```

Comment:
Is this the check that the layer is a volume tracing layer: `layerInfos.tracingId != null`?

Reply:
Yes, exactly. `layerInfos.tracingId` is only defined for volume tracing layers (either with or without fallback). I added a comment to explain the three disabled-reasons. Instead of `tracingId`, I could also add another property, such as `isVolumeTracing`, but that would be somehow redundant, which is why I wasn't a big fan of that :thinking: